packages/vue/src/components/organisms/SfMegaMenu/SfMegaMenu.stories.js

```diff
@@ -2,343 +2,373 @@ import { storiesOf } from "@storybook/vue";
-          :title="subcategory.header"
-        >
-          <SfList>
-            <SfListItem v-for="item in subcategory.items" :key="item.label" >
-              <SfMenuItem
-                :label="item.label" 
-              />
-            </SfListItem>
-          </SfList>
-        </SfMegaMenuColumn>
-      </SfMegaMenu>`,
+        <SfList>
+          <SfListItem v-for="(subcategory, key) in category.subcategories">
+            <SfMenuItem :label="subcategory.title"></SfMenuItem>
+          </SfListItem>
+        </SfList>
+      </SfMegaMenuColumn>
+      <SfMegaMenuColumn title="Featured" class="sf-mega-menu-column--pined-content-on-mobile sf-mega-menu-column--hide-header-on-mobile sb-mega-menu__promo">
+        <div class="sb-mega-menu__banners">
+          <SfBanner
+            v-for="(banner, key) in banners"
+            :key="key"
+            :title="banner.title"
+            :subtitle="banner.subtitle"
+            :image="banner.pictures"
+            class="sb-mega-menu__banner"
+          />
+        </div>
+      </SfMegaMenuColumn>
+    </SfMegaMenu>`,
   }))
-  .add("[slot] aside", () => ({
+  .add("Search results", () => ({
     components: {
       SfMegaMenu,
-      SfMenuItem,
       SfList,
-      AsidePlaceholder,
+      SfBanner,
+      SfProductCard,
+      SfScrollable,
+      SfMenuItem,
     },
-    props: {
-      title: {
-        default: text("title", "Man", "Props"),
-      },
-      asideTitle: {
-        default: text("asideTitle", "Featured", "Props"),
-      },
+    data() {
+      return {
+        categories: [
+          "Long dresses",
+          "Summer dresses",
+          "Dresses",
+          "Coctail dresses",
+          "New dresses",
+          "Dresses on SALE",
+        ],
+      };
+    },
+    template: `<SfMegaMenu
+        :visible="true"
+        title="Search results"
+        class="sb-mega-menu"
+    >
+      <SfMegaMenuColumn title="Categories" class="sf-mega-menu-column--pined-content-on-mobile" style="flex: 0 0 220px; --menu-item-mobile-nav-icon-display: none;">
+        <SfList>
+          <SfListItem v-for="(category, key) in categories">
+            {{category}}
+          </SfListItem>
+        </SfList>
+      </SfMegaMenuColumn>
+      <SfMegaMenuColumn title="Product suggestions" class="sf-mega-menu-column--pined-content-on-mobile" style="flex: 1">
+        <template #title="{title}">
+          <div style="display: flex; align-items: center; --menu-item-mobile-nav-icon-display: none;">
+            <SfMenuItem
+                :label="title"
+                class="sf-mega-menu-column__header"
+            />
+            <span style="margin: 0 0 0 var(--spacer-sm); color:var(--c-gray); font: var(--font-normal) var(--font-sm)/1.4 var(--font-family-secondary);">(20 items)</span>
+          </div>
+        </template>
+        <SfScrollable style="--scrollable-max-height: 630px;" show-text="" hide-text="">
+          <div style="display: flex; flex-wrap: wrap;">
```

Comment:
Can we center search results? 
![image](https://user-images.githubusercontent.com/34887212/82661453-56625380-9c2c-11ea-935c-810752e505a6.png)
or make 2 columns as in designs:
![image](https://user-images.githubusercontent.com/34887212/82661510-7560e580-9c2c-11ea-89db-177ef38edbeb.png)

Reply:
Yes, but IMO we can do it on another issue :)